bindings/go/evmc/evmc.go

```diff
@@ -391,6 +394,12 @@ func (vm *VMSteppable) StepN(params StepParameters) (res StepResult, err error)
 		stack = (*C.evmc_uint256be)(unsafe.Pointer(&params.Stack[0]))
 	}
 
+	pnr := new(runtime.Pinner)
```

Comment:
The `runtime.Pinner` type only got added in go 1.21. This should be reflected by updating the minimum Go version in the `go.mod` file of this project.